tests/cli/functional/actions/test_config_actions.py

```diff
@@ -79,25 +79,30 @@ def config(request, mocker):
     mocker.resetall()  # dont carry over context between functions
 
 
-def test_forget_cli_action(alice_blockchain_test_config, test_emitter, stdout_trap, mock_click_confirm, mocker):
+def test_forget_cli_action(alice_blockchain_test_config, test_emitter, mock_stdin, mocker, capsys):
     mock_forget = mocker.patch.object(CharacterConfiguration, 'forget_nodes')
-    mock_click_confirm.return_value = YES
+    mock_stdin.line(YES)
     forget(emitter=test_emitter, configuration=alice_blockchain_test_config)
     mock_forget.assert_called_once()
+    assert mock_stdin.empty()
+    captured = capsys.readouterr()
+    assert "Permanently delete all known node data?" in captured.out
+    assert "Removed all stored known nodes metadata and certificates" in captured.out
 
 
-def test_update_configuration_cli_action(config, test_emitter, stdout_trap, test_registry_source_manager):
+def test_update_configuration_cli_action(config, test_emitter, test_registry_source_manager, capsys):
     config_class, config_file = config.__class__, config.filepath
     updates = dict(federated_only=True)
     get_or_update_configuration(emitter=test_emitter, config_class=config_class, filepath=config_file, updates=updates)
     config.update.assert_called_once_with(**updates)
     configure.handle_invalid_configuration_file.assert_not_called()
     configure.handle_missing_configuration_file.assert_not_called()
+    captured = capsys.readouterr()
+    assert "Updated configuration values: federated_only" in captured.out
```

Comment:
I think you can use `SUCCESSFUL_UPDATE_CONFIGURATION_VALUES` from `cli.literature` here?

Reply:
Fixed.